src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -1176,7 +1176,9 @@ private BuildEventArgs ReadPropertyReassignmentEventArgs()
                 propertyName,
                 previousValue,
                 newValue,
-                location,
+                fields.File,
```

Comment:
In the Reader we can't change the logic without also checking the version. Because if we're reading an old binlog, this is going to read corrupt data.

You can add if (version >= 25) ... read new stuff, otherwise leave the current logic

Reply:
You changed the reader to read the File, Line and Column separately, but the writer still writes Location as a single string